join_github_app/templates/pages/digital-justice-user.html

```diff
@@ -0,0 +1,48 @@
+{% extends "components/base.html" %}
+
+{% block pageTitle %}
+Using SSO for GitHub Access
+{% endblock %}
+
+{% block beforeContent %}
+{{ govukBackLink ({
+'text': 'Back',
+'href': '/select-organisations'
+}) }}
+{% endblock %}
+
+{% block content %}
+<h1 class="govuk-heading-l">Using Single Sign-On Access</h1>
+
+<p class="govuk-body">
+  Your email address <strong>{{email}}</strong> is registered for SSO access to the GitHub organisation. Single Sign-On
```

Comment:
```suggestion
  Your email address <strong>{{email}}</strong> is registered for Single Sign-On (SSO) access to the GitHub organisations below. SSO
```
Abbreviate after the first instance